### ISSUE TITLE
Adapt to React 19

### ISSUE THREAD

**Issue by ngusta** [glennflanagan/react-collapsible#256]
**Describe the bug**
Requires React <= 18 and 19 is now released.

**To Reproduce**
Upgrade app to React 19, and the dependency check will now break.

**Expected behavior**
It works on React 19, so I'd like to not get the dependency issue,

**Comment by glennflanagan** [comment]
Does the library not work with React 19 or is it just npm/yarn/pnpm reporting the version difference? If you add an override to your package.json does it work?

**Comment by ngusta** [comment]
Yep, the library works fine so it's just npn complaining about the version
difference.

/Niklas

**Comment by onclivia** [comment]
Fixed it by adding this in package.json object:

```
"overrides": {
        "react-collapsible": {
            "react": "^19.0.0",
            "react-dom": "^19.0.0"
        }
    }
```
